helm/cluster-api/values.yaml

```diff
@@ -1,29 +1,19 @@
 name: cluster-api
-
 registry:
   domain: quay.io
-
 images:
   core:
     name: giantswarm/cluster-api-controller
   bootstrap:
     name: giantswarm/kubeadm-bootstrap-controller
   controlplane:
     name: giantswarm/kubeadm-control-plane-controller
-  tag: v0.4.1
-
-featuregates:
-  machinepool: true
-  clusterresourceset: true
-  clustertopology: true
-
-ports:
-  healthz: 9440
-  metrics: 8080
-  webhook: 9443
+  tag: v0.3.22-gs3
```

Comment:
This is currently deployed to our AWS MCs, so we keep the same version for now in order to reduce the scope of changes. We will upgrade to v1.x.x immediately after releasing and deploying this.